Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1040,10 +1044,15 @@ def handle_submitted_notable(service, notable, enrichment_timeout):
             if enrichment.status == Enrichment.IN_PROGRESS:
                 try:
                     job = client.Job(service=service, sid=enrichment.id)
-                    if job.is_ready():
+                    if job.is_done():
                         demisto.debug('Handling open {} enrichment for notable {}'.format(enrichment.type, notable.id))
-                        for item in results.ResultsReader(job.results()):
+                        is_item_fnd = False
+                        job_results = job.results()
+                        for item in results.ResultsReader(job_results):
+                            is_item_fnd = True
                             enrichment.data.append(item)
+                        if not is_item_fnd:
+                            demisto.debug('job.results={}'.format(job_results))
```

Comment:
as the `job.results()` are object - log it without any additional info it will not have meaningful
i think you can ignore it and just write some informative details to the log in that case